include/libswiftnav/track.h

```diff
@@ -146,8 +146,9 @@ typedef struct {
   double carrier_freq;     /**< Carrier frequency in Hz. */
   u32 time_of_week_ms;     /**< Number of milliseconds since the start of the
                                 GPS week corresponding to the last code rollover.  */
-  double receiver_time;    /**< Receiver clock time at which this measurement
-                                is valid in seconds. */
+  double rec_time_delta;   /**< Difference between receiver clock time at which
```

Comment:
hey, to fix the current travis build failure, I think you just need to carry over this update here:
https://github.com/swift-nav/libswiftnav/blob/master/python/swiftnav/track.pxd#L166